docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,219 @@
+# Create Your First Tizen IoT .NET Application
```

Comment:
This IoT Page is based on the other existing profile pages.
ex) https://docs.tizen.org/application/dotnet/get-started/mobile/first-app/

I think they should have a similar format.

Reply:
@xbldx Thank you for your explanations. 
@mijin85cho  request your valuable opinion on the same.

Reply:
As I can understand, this can be a Tizen .Net based IoT app for any profile like wearable, Tv, etc.

As per my understanding, (which can be wrong ) the apps whatever we develop, will be on its core will be Tizen.NET based. These apps can, for instance, be called as **Tizen .Net based wearable/mobile/Tv IoT app**. Here the utilization of the developed apps will address the IoT use case.

Reply:
It's not true. Tizen IoT profile is for IoT devices.

Reply:
@xbldx  thanks for your reply. 
Cant an IoT app run on a wearable?
- I mean isn't a wearable considered as IoT device? 
- I may be wrong, but you can help me understand this.
   https://ieeexplore.ieee.org/document/9058658
   https://www.google.com/search?client=firefox-b-d&q=are+wearables+part+of+iot

Reply:
Your question is a little ambiguous. 
Can you launch wearable app on a mobile? It is Yes or No.

There are Mobile, Wearable, TV and IoT profiles in Tizen.
It can be changed later. But IoT is one of Tizen profile now.

Reply:
Our wearable profile is not related to IoT profile directly.
In other words, our IoT profile is not for wearable(watch) devices now.